webui/src/extension-registry-types.ts

```diff
@@ -190,6 +191,12 @@ export interface Namespace {
     roleUrl: UrlString;
 }
 
+export interface PublisherInfo {
+    user: UserData;
+    extensions: Extension[];
+    accessTokens: PersonalAccessToken[];
```

Comment:
Are the access tokens required here at all? Maybe we could get rid of them and just send the user data and extensions.